JavaAssignment/src/piyushS/Q2.java

```diff
@@ -0,0 +1,23 @@
+package piyushS;
+
+public class Q2 {
+
+	void detailsOfStudent(int RollNo, String Name, int Marks) {
+
+		System.out.println("Rollnumber of a Student is " + RollNo);
+
+		System.out.println("Name of a Student is " + Name);
+
+		System.out.println("Marks of a Student is " + Marks);
+
+	}
+
+	public static void main(String[] args) {
+
+		Q2 obj = new Q2();
+
+		obj.detailsOfStudent(101, "PiyushS", 500);
```

Comment:
In this program, we have to take run time value  from user. from args[]. Please do this change.